### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dhiru30**
Corrected the grammar of the document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ make clean test
 ```
 
 > Note: if you are running on Windows,
-[this article includes important notes for compiling Elixir from source
+[This article includes important notes for compiling Elixir from source
```
Comment (josevalim) [inline README.md]: Since the previous character is a comma, we should keep lowercase here.

File: README.md [hunk 2]
```diff
@@ -133,7 +133,7 @@ We outline our process below to clarify the roles of everyone involved.
 
 All pull requests must be approved by two committers before being merged into
 the repository. If any changes are necessary, the team will leave appropriate
-comments requesting changes to the code. Unfortunately we cannot guarantee a
+comments requesting changes to the code. Unfortunately, we cannot guarantee a
```
Comment (josevalim) [inline README.md]: According to [this discussion](https://english.stackexchange.com/questions/418536/comma-after-unfortunately-in-the-beginning-of-a-sentence), the comma does not seem to be required.

File: README.md (hunk 1 above)
Comment (dhiru30) [reply README.md]: @josevalim  Yes correct. I will revert it.

File: README.md (hunk 2 above)
Comment (dhiru30) [reply README.md]: @josevalim Ok I will close the pull request.

**Pull request closed without merging**